### PR TITLE
SDK-241 [Java SDK] Improve integration tests

### PR DESCRIPTION
**Pull request by robert-par**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-241](https://incountry.atlassian.net/browse/SDK-241)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version? What is the new version?

### CODE REVIEW

**Review by RommeroInCountry — DISMISSED** [review]
LGTM

File: src/integration/java/com/incountry/residence/sdk/StorageIntegrationTest.java [hunk 1]
```diff
@@ -126,11 +127,11 @@
     private static final String COUNTRIES_LIST_ENDPOINT = loadFromEnv(INT_COUNTRIES_LIST_ENDPOINT);
 
     private static final int VERSION = 0;
-    private static final String FILE_CONTENT = "Hello world!";
+    private static final String FILE_CONTENT = UUID.randomUUID().toString();
     private static final String DEFAULT_MIME_TYPE = "multipart/form-data";
     private static final String NEW_FILE_NAME = "new_sdk_incountry_test_file";
```
Comment (VladimirLevitskiy) [inline src/integration/java/com/incountry/residence/sdk/StorageIntegrationTest.java:132]: Use UUID for NEW_FILE_NAME too

File: src/integration/java/com/incountry/residence/sdk/StorageIntegrationTest.java [hunk 2]
```diff
@@ -212,7 +213,7 @@ public void writeTest() throws StorageException {
 
     @SuppressWarnings("java:S2696")
```
Comment (VladimirLevitskiy) [inline src/integration/java/com/incountry/residence/sdk/StorageIntegrationTest.java:214]: remove this SuppressWarnings and fix it's requirments

File: src/integration/java/com/incountry/residence/sdk/StorageIntegrationTest.java [hunk 3]
```diff
@@ -312,73 +315,86 @@ void updateAttachmentMetaTest() throws StorageException {
...
         AttachmentMeta meta = storage.getAttachmentMeta(MIDIPOP_COUNTRY, RECORD_KEY, UUID.randomUUID().toString());
         assertNull(meta);
     }
 
     @Test
-    @Order(311)
+    @Order(359)
     public void updateAttachmentMetaForUnexistingFileTest() {
         StorageServerException ex = assertThrows(StorageServerException.class, () -> storage.updateAttachmentMeta(MIDIPOP_COUNTRY, RECORD_KEY, UUID.randomUUID().toString(), NEW_FILE_NAME, MIME_TYPE));
         assertTrue(ex.getMessage().contains("Code=404"));
```
Comment (VladimirLevitskiy) [inline src/integration/java/com/incountry/residence/sdk/StorageIntegrationTest.java:382]: It is possible to use more text at this check?

File: src/integration/java/com/incountry/residence/sdk/StorageIntegrationTest.java [hunk 4]
```diff
@@ -312,73 +315,86 @@ void updateAttachmentMetaTest() throws StorageException {
...
     }
 
+    @SuppressWarnings("java:S2696")
```
Comment (VladimirLevitskiy) [inline src/integration/java/com/incountry/residence/sdk/StorageIntegrationTest.java:385]: remove this SuppressWarnings too

**Review by VladimirLevitskiy — CHANGES_REQUESTED** [review]

**Review by VladimirLevitskiy — APPROVED** [review]
LGTM

**Review by RommeroInCountry — APPROVED** [review]
LGMT

**Pull request merged**